portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,35 @@ function addRandomFunFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+/**
+ * Returns a greeting in different languagues randomly.
+ */
+function getRandomGreeting() {
+    const greetings =
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+
+    // Pick a random greeting.
+    var greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    return greeting;
+}
+
+/**
+ * Prints the given array in a readable string format.
+ */
+function printInfoAboutMe(myInfo){
+  let index;
+  let initialInfo = "";
+  for(index = 0; index < myInfo.length; index++){
```

Comment:
you can inline the "let index" 
So you can remove "let index" on line 45, and instead have 
for(let index=0; index......)